code/renderers/react/template/components/Form.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export const Form = ({ onSuccess }) => {
+  const [value, setValue] = useState('');
+  const [complete, setComplete] = useState(false);
+
+  function onSubmit(event) {
+    event.preventDefault();
+    onSuccess(value);
+
+    setTimeout(() => setComplete(true), 500);
+    setTimeout(() => setComplete(false), 1500);
```

Comment:
What's the idea here @tmeasday ?

Reply:
The "Success" information disappears, so we can test `WaitForElementToBeRemoved`